spec/helpers/check_answer_url_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe CheckAnswerUrlHelper, type: :helper do
+  let(:application) { build :legal_aid_application, id: 'bf07c530-718a-4537-b9e1-c04ad956cc71' }
+
+  describe '#check_answer_url_for' do
+    context 'provider' do
+      it 'returns the correct path' do
+        url = check_answer_url_for(:provider, :outstanding_mortgage, application)
+        expect(url).to eq '/providers/applications/bf07c530-718a-4537-b9e1-c04ad956cc71/outstanding_mortgage#outstanding_mortgage_amount'
```

Comment:
I'd suggest not specifying the id in `let(:application)` and then:
```ruby
expect(url).to eq "/providers/applications/#{application.id}/outstanding_mortgage#outstanding_mortgage_amount"
```

Reply:
I did it this way because I think it makes it a bit clearer, but will change if you think it's important

Reply:
I'd argue, that it's easier to see that `application.id` is the same as `application.id` than it is to see that `bf07c530-718a-4537-b9e1-c04ad956cc71` is not the same as `bf07c530-719a-4537-b9e1-c04ad956cc71`